python/pylibcudf/pylibcudf/strings/split/partition.pyx

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+from libcpp.memory cimport unique_ptr
+from libcpp.utility cimport move
+from pylibcudf.column cimport Column
+from pylibcudf.libcudf.scalar.scalar cimport string_scalar
+from pylibcudf.libcudf.strings.split cimport partition as cpp_partition
+from pylibcudf.libcudf.table.table cimport table
+from pylibcudf.scalar cimport Scalar
+from pylibcudf.table cimport Table
+
+from cython.operator import dereference
+
+
+cpdef Table partition(Column input, Scalar delimiter):
```

Comment:
Can we add the empty string default value here? We've done it like [this](https://github.com/rapidsai/cudf/blob/branch-24.12/python/pylibcudf/pylibcudf/strings/strip.pyx#L44-L47) so far.

Reply:
Ah OK. I think I may have missed this default value pattern to match the libcudf default value in my recently pylibcudf PRs. I'll got back in a follow up to try to address this.